env.arolla.sh

```diff
@@ -349,7 +349,12 @@ EOF
     export CC=gcc
 
     # CLAW Compiler using the correct preprocessor
-    export CLAWFC="${installdir}/claw/v2.0.1/${compiler}/bin/clawfc"
+    if [ "${compiler}" == "pgi" ]; then
+        export CLAWFC="${installdir}/claw/v2.0.1/${compiler}/bin/clawfc"
+    else
+        # CLAW v2.0.1 only works with PGI atm
```

Comment:
Is this also the case on kesch ? I merged it like this. But we probably should change the if condition